code/game/objects/items/dualsaber.dm

```diff
@@ -63,6 +63,9 @@
 	set_light_on(TRUE)
 
 
+/obj/item/dualsaber/add_blood_DNA(list/blood_DNA_to_add)
+	return FALSE
```

Comment:
This should probably be an object flag like `NO_BLOODY`

Reply:
would be nice if someone did it, however I'm not expecting it to be done out of this PR because it's a minor fix for an issue report that makes it consistent with the already existing brother, the esword.